ofrak_core/pytest_ofrak/patterns/basic_block_unpacker.py

```diff
@@ -1319,6 +1319,36 @@ class BasicBlockUnpackerTestCase(
             0x4005A0: Range(0x5A0, 0x5AF),
         },
     ),
+    # x64 Kernel address space
```

Comment:
Please update this comment to more explicitly state what bug/failure it is testing (see proposed changelog).

As a sanity check -- this test should **_FAIL_** without the change. Is this the case?

Reply:
Yes, I've confirmed that these tests fail without the Ghidra script changes.